python/equistore-operations/tests/manipulate_labels_column.py

```diff
@@ -80,7 +80,7 @@ def test_append_properties():
 
 
 def test_append_unknown_axis():
-    with pytest.raises(ValueError, match="foo is not a valid axis."):
+    with pytest.raises(ValueError, match="'foo' is not a valid axis."):
```

Comment:
could you move the file to `manipulate_dimensions`?

Reply:
Yes sure. I don't know why I didn't...